tests/results/graceful-recover-results.md

```diff
@@ -0,0 +1,15 @@
+# Test Results
```

Comment:
I'd mention any issues you opened here -- because they provide even more details about the encountered problems